Dockerfile

```diff
@@ -38,4 +39,5 @@ USER caikit
 ENV CONFIG_FILES=/caikit/config/caikit.yml
 VOLUME ["/caikit/config/"]
 
+ENTRYPOINT ["/caikit/wait-modelcar.sh"]
 CMD ["python",  "-m", "caikit.runtime"]
```

Comment:
What are the semantics here of changing the entrypoint? Does CMD run afterwards? I was under the impression that the entrypoint is what's run when the container starts up

Reply:
When an ENTRYPOINT is specified, then the CMD becomes its arguments (the separation comes from the early Docker days, so that the image author could specify a command with ENTRYPOINT that could not be overwritten with `docker run`). In the Kubernetes world, ENTRYPOINT correspond to the `command:` field and CMD correspond to the `args:` field in a Pod container's spec.